app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationRouter.kt

```diff
@@ -127,7 +127,7 @@ private fun ConversationRouter(
                 composable(
                     route = ConversationsNavigationItem.Calls.route,
                     content = {
-                        CallScreen(
+                        CallsScreen(
```

Comment:
why calls now and not call ?

Reply:
It may be confusing with the other calling screens. 
I know It's not the best name I have chosen. I thought about names like `CallsActivitiesScreen`, `RecentCallsScreen` But I keep it simple with just `CallsScreen` Any suggestions ?

Reply:
could you tell me what this screen will actually do ? maybe I can give some suggestion  :P

Reply:
<img width="289" alt="Screenshot 2022-04-08 at 17 28 26" src="https://user-images.githubusercontent.com/18124919/162483580-1671a233-5e8b-4d3a-8fda-548fb748d8d1.png">